src/Collections/Maps/Maps.dfy

```diff
@@ -93,6 +97,39 @@ module {:options "-functionSyntax:4"} Maps {
     forall x, x' {:trigger m[x], m[x']} :: x != x' && x in m && x' in m ==> m[x] != m[x']
   }
 
+  ghost predicate {:opaque} Contains<X, Y>(big: map<X, Y>, small: map<X, Y>)
```

Comment:
Consider `SubmapOf` (and reversing the arguments) instead, bit more consistent with the other `Collections` code.

Bit unfortunate Dafny doesn't define `<=` for maps. :)

Reply:
Turns out `IsSubset` was already defined, so I've switched everything to use that and removed the redundant `Contains`.